rotkehlchen/db/upgrades/v32_v33.py

```diff
@@ -72,6 +72,10 @@ def _create_new_tables(cursor: 'Cursor') -> None:
 """)
 
 
+def _refactor_blockchain_account_labels(cursor: 'Cursor') -> None:
+    cursor.execute('UPDATE blockchain_accounts SET label = NULL WHERE label = ""')
```

Comment:
The value provided by the frontend will be inserted. Previously it was `""` but we agreed with @lukicenturi to make it `null`

Reply:
Shouldn't then:

1. the frontend change happen at the same place otherwise we may end up with DBs in incosistent state?
2. check for empty string at a label via the API and return an error?

Reply:
1. Yeah, should
2. Makes sense, will do it